views/files_layout.erb

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
```

Comment:
I want to figure out how to nest layouts, so this file should extend `views/layout.erb`.